sr_blockly_blocks/generators/python/scripts/initialise.py

```diff
@@ -9,6 +9,8 @@
     hand_commander = SrHandCommander(name=side + "_hand")
 elif dropdown_commander == 'robot_commander':
     robot_commander = SrRobotCommander(name=side + "_arm_and_hand")
+elif dropdown_commander == 'hand_h_and_arm':
+    robot_commander = SrRobotCommander ("right_arm_and_hand", prefix="H1_")
```

Comment:
But that is actually a good point. We will need to modify the `side` dropdown so that it makes sense for hand H as well.